src/Core/Grid/Definition/Factory/CountryGridDefinitionFactory.php

```diff
@@ -0,0 +1,254 @@
+                        'required' => false,
+                        'attr' => [
+                            'placeholder' => $this->trans('Search ID', [], 'Admin.Actions'),
+                        ],
+                    ])
+            )
+            ->add(
+                (new Filter('name', TextType::class))
+                    ->setAssociatedColumn('name')
+                    ->setTypeOptions([
+                        'required' => false,
+                        'attr' => [
+                            'placeholder' => $this->trans('Search country', [], 'Admin.Actions'),
+                        ],
+                    ])
+            )
+            ->add(
+                (new Filter('iso_code', TextType::class))
+                    ->setAssociatedColumn('iso_code')
+                    ->setTypeOptions([
+                        'required' => false,
+                        'attr' => [
+                            'placeholder' => $this->trans('Search ISO code', [], 'Admin.Actions'),
+                        ],
+                    ])
+            )
+            ->add(
+                (new Filter('call_prefix', TextType::class))
+                    ->setAssociatedColumn('call_prefix')
+                    ->setTypeOptions([
+                        'required' => false,
+                        'attr' => [
+                            'placeholder' => $this->trans('Search call prefix', [], 'Admin.Actions'),
+                        ],
+                    ])
+            )
+            ->add(
+                (new Filter('zone_name', TextType::class))
+                    ->setAssociatedColumn('zone_name')
+                    ->setTypeOptions([
+                        'required' => false,
+                        'attr' => [
+                            'placeholder' => $this->trans('Search zones', [], 'Admin.Actions'),
+                        ],
+                    ])
+            )
+            ->add(
+                (new Filter('active', YesAndNoChoiceType::class))
+                    ->setAssociatedColumn('active')
+                    ->setTypeOptions([
+                        'required' => false,
+                        'choice_translation_domain' => false,
+                    ])
+            )
+            ->add(
+                (new Filter('actions', SearchAndResetType::class))
+                    ->setAssociatedColumn('actions')
+                    ->setTypeOptions([
+                        'reset_route' => 'admin_common_reset_search_by_filter_id',
+                        'reset_route_params' => [
+                            'filterId' => self::GRID_ID,
+                        ],
+                        'redirect_route' => 'admin_countries_index',
+                    ])
+            );
+    }
+
+    /**
+     * @return RowActionCollection
+     */
+    private function getRowActions(): RowActionCollection
+    {
+        $rowActionCollection = new RowActionCollection();
```

Comment:
You need to add the row actions here, even if the form part is not implemented what we usually do is create the routes and controllers. Only the controllers are very basic they simply redirect to the legacy page of the form.

Reply:
Please add a todo here, the row actions must be added at some point even if the form is not done we can create controllers that simply redirect to legacy page